packages/hint-x-content-type-options/src/hint.ts

```diff
@@ -78,7 +79,7 @@ export default class XContentTypeOptionsHint implements IHint {
                 return;
             }
 
-            const headerValue: string = normalizeString(response.headers && response.headers['x-content-type-options']);
+            const headerValue: string = normalizeString(response.headers && response.headers['x-content-type-options']) || '';
```

Comment:
This needs to be `headerValue: string | null` instead of falling back to the empty string to avoid altering behavior (note the explicit check for `if (headerValue === null)` a few lines later. I'll update the PR shortly.